Stripe/STPSource+Private.h

```diff
@@ -8,11 +8,22 @@
 
 #import "STPSource.h"
 
-@interface STPSource (Private)
-+ (NSString *)stringFromType:(STPSourceType)type;
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPSource ()
+
 + (STPSourceType)typeFromString:(NSString *)string;
-+ (NSString *)stringFromFlow:(STPSourceFlow)flow;
-+ (NSString *)stringFromUsage:(STPSourceUsage)usage;
-@end
++ (nullable NSString *)stringFromType:(STPSourceType)type;
+
++ (STPSourceFlow)flowFromString:(NSString *)string;
++ (nullable NSString *)stringFromFlow:(STPSourceFlow)flow;
 
++ (STPSourceStatus)statusFromString:(NSString *)string;
++ (nullable NSString *)stringFromStatus:(STPSourceStatus)status;
+
++ (STPSourceUsage)usageFromString:(NSString *)string;
++ (nullable NSString *)stringFromUsage:(STPSourceUsage)usage;
```

Comment:
Changed a bunch of these `stringFrom*` to be `nullable` here to account for returning `nil` when the enum value is `*Unknown`